app/views/administrate/application/_collection.html.erb

```diff
@@ -18,13 +18,13 @@ to display a collection of resources in an HTML table.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Table
 %>
 
-<table class="collection-data">
+<table class="collection-data" aria-labelledby="page-title">
```

Comment:
Very interesting…

I have mixed feelings, though, because an example heading (which is what I’m tying the ARIA attribute to) for these pages is some thing like “Customers”. So it’s not really a “summary”, per se, and according to the description from the W3:

> A summary conveys information about the organization of the data in a table and helps users navigate it.